neqo-common/src/qlog.rs

```diff
@@ -93,6 +93,10 @@ impl NeqoQlog {
         F: FnOnce(&mut QlogStreamer) -> Result<(), qlog::Error>,
     {
         if let Some(inner) = self.inner.borrow_mut().as_mut() {
+            // TODO: Hack.
+            if inner.qlog_path == Path::new("") {
+                return;
+            }
```

Comment:
@KershawChang said in Slack `self.inner` should be `None` in the case where `self` is `NeqoQlog::disabled()`, but that doesn't seem to be the case?

But we need a better check here than the current hack above.